### DISCUSSION TITLE
Can we do this using Host?

### DISCUSSION THREAD

**Discussion by JamesTanner651** [AutomaApp/automa#437] [General]
is it possible to sync all workflows in different devices but keeping google sheet ranges different?

I have one workflow which I can sync with all the devices but it also sync google sheet ranges and the task is not unique anymore on them. 

How to go about this?

**Comment by Kholid060** [comment]
Maybe I can enable edit the [global data](https://docs.automa.site/api-reference/global-data.html#global-data), so you can store the spreadsheet id and sheet id in the global data

![image](https://user-images.githubusercontent.com/22908993/162648394-0e4bbe0f-701b-4240-a7fb-8e199def5be2.png)

And in the google sheets block, you can do like in the image

![image](https://user-images.githubusercontent.com/22908993/162648466-f00d8ce5-6702-4689-a013-5873425ee92d.png)

**Reply by JamesTanner651** [reply]
I wont be able to run those workflows simultaneously that way right?
How can I resume loop from where it ended in next execution? this way I'll be able to keep ranges the same but Automa will resume from last value

**Reply by Kholid060** [reply]
> I won't be able to run those workflows simultaneously that way right?

Yes, but if you want to execute those workflows simultaneously you can set the trigger to `on a specific day` or `on a specific date`

> How can I resume loop from where it ended in next execution? 

In the loop data block, enable the `Resume last workflow` option.

![image](https://user-images.githubusercontent.com/22908993/162854807-13124045-edee-4e88-b17e-3b50f28529f8.png)

**Reply by JamesTanner651** [reply]
Hey. I tried resume last workflow, it didn't work. It started from the first value again. 
I had 10 values and I set the maximum loop limit to 3. It should've started from 4 but it again start from 1.

**Reply by Kholid060** [reply]
It may be a bug I'll look into this

**Reply by JamesTanner651** [reply]
> It may be a bug I'll look into this

Will you be fixing this bug in the upcoming version?
